src/ig/IGExporter.ts

```diff
@@ -383,24 +383,25 @@ export class IGExporter {
         exampleBoolean: false
       });
     });
-    const examples = sortBy(
+    const instances = sortBy(
       this.pkg.instances,
       instance => instance.id ?? instance._instanceMeta.name
     );
-    examples.forEach(example => {
+    instances.forEach(instance => {
       const resource: ImplementationGuideDefinitionResource = {
         reference: {
-          reference: `${example.resourceType}/${example.id ?? example._instanceMeta.name}`
+          reference: `${instance.resourceType}/${instance.id ?? instance._instanceMeta.name}`
         },
-        name: example._instanceMeta.title ?? example._instanceMeta.name,
-        description: example._instanceMeta.description
+        name: instance._instanceMeta.title ?? instance._instanceMeta.name,
+        description: instance._instanceMeta.description
       };
-      const exampleUrl = example.meta?.profile?.find(url => this.pkg.fish(url, Type.Profile));
-      if (exampleUrl) {
-        resource.exampleCanonical = exampleUrl;
-      } else {
-        resource.exampleBoolean =
-          example._instanceMeta.usage === 'Example' || example._instanceMeta.usage === undefined;
+      if (instance._instanceMeta.usage === 'Example') {
+        const exampleUrl = instance.meta?.profile?.find(url => this.pkg.fish(url, Type.Profile));
+        if (exampleUrl) {
+          resource.exampleCanonical = exampleUrl;
+        } else {
+          resource.exampleBoolean = true;
+        }
```

Comment:
I think this needs one more tweak. I think that `resource.exampleBoolean` should be set to `false` when the instance is not an example at all. We set `exampleBoolean` to `false` on SDs, ValueSets, and CodeSystems (when they are made using the keywords in FSH), so I think we should also set it to `false` on things like `CapabilityStatement`s and so on.

It's possible this doesn't really matter. Maybe the IG publisher reads not setting `exampleBoolean` the same as setting it to `false`, but we should be consistent to be safe.

Reply:
I'm good with being consistent. I was just following what we used to have, but I like consistency better!